### PR TITLE
meeting minutes Jan 18, 2023

### PR DESCRIPTION
**Pull request by vickimzhang**
can u believe it's 2023?????

### CODE REVIEW

**Comment by aays** [comment]
Looks good, thanks so much! Could the file just be renamed to `Minutes_2023-01-08.md`? Would also be good to have an attendance list at the top, e.g. `Present: ..., Regrets: ...` - see [here](https://github.com/UofTCoders/council/blob/master/meetings/Minutes_2022-12-05.md) for an example from last meeting.

**Comment by vickimzhang** [comment]
> Looks good, thanks so much! Could the file just be renamed to `Minutes_2023-01-08.md`? Would also be good to have an attendance list at the top, e.g. `Present: ..., Regrets: ...` - see [here](https://github.com/UofTCoders/council/blob/master/meetings/Minutes_2022-12-05.md) for an example from last meeting.

Done! Saved to the branch, ready for the PR.

File: meetings/Minutes_2023-01-08.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Present: Vicki, Yeshoda, Madeleine, Sophie, Ahmed, Elliot;
```
Comment (QuLogic) [inline meetings/Minutes_2023-01-08.md:1]:
```suggestion
Present: Vicki, Yeshoda, Madeleine, Sophie, Ahmed, Elliott;
```

File: meetings/Minutes_2023-01-08.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+Love EasyRetro -  more responses than previous surveys (i.e., 4 ticks on some things)
+- difficult to check how many people total responded, but results below reflect survey results
+
+### Lessons
+
+Survey
+
+- Python relatively more popular
+    - data vis with Python, intro to bas, Numpy & Pandas, Machine Learning
```
Comment (QuLogic) [inline meetings/Minutes_2023-01-08.md:32]:
```suggestion
    - data vis with Python, intro to bash, NumPy & Pandas, Machine Learning
```

File: meetings/Minutes_2023-01-08.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    - data vis with R also popular, but ran it last semester so skipping it this time
+- no one put down new lessons
+
+Weekly or reduce to bi-weekly and/or make lessons longer?
+
+- stick to 4 lessons, maybe 5 if there’s another interesting topic
+    - data vis with Python, intro to bash, Numpy & Pandas, Intro to Machine Learning
```
Comment (QuLogic) [inline meetings/Minutes_2023-01-08.md:39]:
```suggestion
    - data vis with Python, intro to bash, NumPy & Pandas, Intro to Machine Learning
```

File: meetings/Minutes_2023-01-08.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Got very few responses about this
+
+- timing didn’t work for one respondent (interesting as there were 2 different times)
+- one responded not interested in co-working sessions
+- but! someone asked for support outside of workshop hours … but … that’s literally … co-working. confusion
+
+Given these results, should we continue with co-working?
+
+- Sophie: no for this semester. However, to promote co-working in the future or on the Discord, could publish more information on the website about how to ask for support/help on the Discord
+    - Ask Elliot about putting Discord information more front & centre (e.g., “Click here for help!”)
```
Comment (QuLogic) [inline meetings/Minutes_2023-01-08.md:63]:
```suggestion
    - Ask Elliott about putting Discord information more front & centre (e.g., “Click here for help!”)
```

**Review by QuLogic — COMMENTED** [review]
Small typos

File: meetings/Minutes_2023-01-08.md (hunk 1 above)
Comment (vickimzhang) [reply meetings/Minutes_2023-01-08.md:1]: sorry for spelling your name wrong so many times :(

**Pull request merged**